configure.ac

```diff
@@ -2597,6 +2597,9 @@ if test "${enable_ebpf}" = "yes" || test "${enable_nfqueue}" = "yes" || test "${
   AC_DEFINE([CAPTURE_OFFLOAD], [1],[Building flow capture bypass code])
 fi
 
+# Add diagnostic filename
+CPPFLAGS="${CPPFLAGS} -D__SCFILENAME__=\\\"\$(*F)\\\""
```

Comment:
Why is this needed instead of just using __FILE__?

Reply:
`__FILE__` is the name of the file currently being processed by cpp whereas `__SCFILENAME__` is the name of the file being compiled. 

This is leveraged in `src/util-debug.h`
```
static const char *_sc_module __attribute__((unused)) = __SCFILENAME__;
```